app/src/main/java/com/android/sample/ui/activity/CreateActivity.kt

```diff
@@ -381,7 +381,9 @@ fun CreateActivityScreen(
                 enabled = title.isNotEmpty() && description.isNotEmpty() && dueDate.isNotEmpty(),
                 onClick = {
                   val timeFormat = startTime.split(":")
-                  if (timeFormat.size != 2) {
+                  val timePattern = Regex("^([01]?[0-9]|2[0-3]):[0-5][0-9]$")
+
+                  if (timeFormat.size != 2 || !timePattern.matches(startTime)) {
                     Toast.makeText(
```

Comment:
It is a great thing that you chose to use a pattern for time. These a more standard and have for guarantee on the checks. Well done.

Reply:
The pattern seems appropriate for the time use case. It is great that it blocks times above 23:59 (such as 27:91).